Formula/a/apitrace.rb

```diff
@@ -0,0 +1,133 @@
+class Apitrace < Formula
+  desc "Tools for tracing OpenGL, Direct3D, and other graphics APIs"
+  homepage "https://apitrace.github.io/"
+  url "https://github.com/apitrace/apitrace/archive/00706d1a395ab6570ab88136fffb3028a3ded3bd.zip"
+  version "11.1+dev.0"
```

Comment:
This is bogus.

I'm using a frankensteined `master`, because I couldn't make Qt5 work and there's no upstream tag for Qt6 (yet).